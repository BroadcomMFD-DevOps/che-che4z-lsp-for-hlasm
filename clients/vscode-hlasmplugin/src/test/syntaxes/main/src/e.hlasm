LABEL INSTR                                                            -12345678
               OP4 OP5,   REMARKS                                      -12345678
               OP6,    R                                               -12346578
               OP7                                                      12345678

.L20     MNOTE                                                         X
               4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
             LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O -12345678
               REMARK REMARK

