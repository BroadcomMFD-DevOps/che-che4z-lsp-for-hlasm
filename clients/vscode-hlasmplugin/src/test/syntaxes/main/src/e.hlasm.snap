>LABEL INSTR                                                            -12345678
#^^^^^ source.hlasm support.class.hlasm
#     ^^^^^^ source.hlasm entity.name.function.hlasm
#           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP4 OP5,   REMARKS                                      -12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm punctuation.definition.tag.hlasm
#                  ^ source.hlasm
#                   ^^^^ source.hlasm comment.hlasm
#                       ^^^ source.hlasm
#                          ^^^^^^^ source.hlasm comment.hlasm
#                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP6,    R                                               -12346578
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^ source.hlasm comment.hlasm
#                   ^^^^ source.hlasm
#                       ^ source.hlasm comment.hlasm
#                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               OP7                                                      12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^ source.hlasm comment.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm invalid.ignored
>
>.L20     MNOTE                                                         X
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
>               4,'Length of block does not exceed 256 (&LBL DSECT), &&CX12345678
#^^^^^^^^^^^^^^^ source.hlasm
#               ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                ^ source.hlasm
#                 ^ source.hlasm string.hlasm
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                       ^^^^ source.hlasm variable.other.readwrite.hlasm
#                                                           ^^^^^^^^^ source.hlasm string.hlasm
#                                                                    ^^ source.hlasm string.hlasm
#                                                                      ^ source.hlasm string.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>             LEANUP option can be optimized to YES',OP2OP2OP2OP2OP2O -12345678
#^^^^^^^^^^^^^^^ source.hlasm invalid.illegal
#               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm string.hlasm
#                                                   ^ source.hlasm
#                                                    ^^^^^^^^^^^^^^^^ source.hlasm punctuation.definition.tag.hlasm
#                                                                    ^ source.hlasm
#                                                                     ^^ source.hlasm comment.hlasm
#                                                                       ^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^ source.hlasm invalid.ignored
>               REMARK REMARK
#^^^^^^^^^^^^^^^ source.hlasm
#               ^^^^^^ source.hlasm comment.hlasm
#                     ^ source.hlasm
#                      ^^^^^^ source.hlasm comment.hlasm
>
>